apps/user-profile/src/state/actions/onboardingActions.js

```diff
@@ -1,14 +1,17 @@
+import { onboardingActions } from ".";
 
-const SHOW_PLATFORM_ONBOARDING_MODAL = "Show platform onboarding modal.";
-const HIDE_PLATFORM_ONBOARDING_MODAL = "Hide platform onbaording modal.";
-const SHOW_INTRODUCTION_MODAL = "Show introduction modal.";
-const HIDE_INTRODUCTION_MODAL = "Hide introduction modal.";
-const HIDE_ALL_MODALS = "Hide All Modals";
-const SET_USERS_ROLE = "Set Users Role";
-const SET_USERS_EXPERIENCE = "Set Users Experience";
-const SET_USERS_INTEREST = "Set Users Interest";
-const REMOVE_USERS_INTEREST = "Remove Users Interest";
-
+const SHOW_PLATFORM_ONBOARDING_MODAL = "SHOW_PLATFORM_ONBOARDING_MODAL";
+const HIDE_PLATFORM_ONBOARDING_MODAL = "HIDE_PLATFORM_ONBOARDING_MODAL";
+const SHOW_INTRODUCTION_MODAL = "SHOW_INTRODUCTION_MODAL";
+const HIDE_INTRODUCTION_MODAL = "HIDE_INTRODUCTION_MODAL";
+const HIDE_ALL_MODALS = "HIDE_ALL_MODALS";
+const SET_USERS_ROLE = "SET_USERS_ROLE";
+const SET_USERS_EXPERIENCE = "SET_USERS_EXPERIENCE";
+const SET_USERS_INTEREST = "SET_USERS_INTEREST";
+const REMOVE_USERS_INTEREST = "REMOVE_USERS_INTEREST";
+const HIDE_CLOSING_MODAL = "HIDE_CLOSING_MODAL";
+const SHOW_CLOSING_MODAL = "SHOW_CLOSING_MODAL";
```

Comment:
Any reason why you switch the values of each variable? e.g 
from:  `const SHOW_PLATFORM_ONBOARDING_MODAL = "Show platform onboarding modal.";` 
to: `const SHOW_PLATFORM_ONBOARDING_MODAL = "SHOW_PLATFORM_ONBOARDING_MODAL";`